bzt/engine.py

```diff
@@ -436,8 +437,14 @@ def _load_user_configs(self, user_configs):
         :type user_configs: list[str]
         :rtype: Configuration
         """
+        # "tab-replacement-spaces" is not documented 'cause it loads only from base configs
+        # so it's sort of half-working last resort
+        self.config.tab_replacement_spaces = self.config.get(SETTINGS).get("tab-replacement-spaces", 4)
```

Comment:
Not a huge fan of using option only from base configs, as it's kind of non-obvious. But as a last resort it's good.